src/programs/solo8ti_hardware_calibration.cpp

```diff
@@ -0,0 +1,65 @@
+/**
+ * \file solo8_hardware_calibration.cpp
+ * \brief ...
+ * \author Maximilien Naveau
+ * \date 2018
+ *
+ * This file uses the Solo8 class in a small demo.
+ */
+
+#include "blmc_robots/common_programs_header.hpp"
+#include "blmc_robots/solo8ti.hpp"
+
+using namespace blmc_robots;
+
+static THREAD_FUNCTION_RETURN_TYPE control_loop(void* robot_void_ptr)
+{
+    Solo8TI& robot = *(static_cast<Solo8TI*>(robot_void_ptr));
+
+    blmc_robots::Vector8d joint_index_to_zero;
+    joint_index_to_zero.fill(0.0);
+    robot.calibrate(joint_index_to_zero);
+
+    long int count = 0;
+    while (!CTRL_C_DETECTED)
+    {
+        if (count % 200 == 0)
+        {
+            robot.acquire_sensors();
+            print_vector("Joint Positions", robot.get_joint_positions());
+        }
```

Comment:
Note that `count` is not increased. So you are doing `count % 200 == 0` all the time.

Also, should there be a wait in the loop?